doc/sphinx/index.rst.in

```diff
@@ -63,4 +62,10 @@ The main documentation for the site is organized into following sections:
    docs/fpga_tools/opaevfio/opaevfio
    docs/fpga_tools/pci_device/pci_device
    docs/fpga_tools/opae.io/opae.io
-   
+   docs/fpga_tools/host_exerciser/host_exerciser
+   docs/fpga_tools/hssi_ethernet/hssistats
+   docs/fpga_tools/hssi_ethernet/hssimac
+   docs/fpga_tools/hssi_ethernet/hssiloopback
+   docs/fpga_tools/opaeuio/opaeuio
```

Comment:
This tool (opaeuio script) no longer exists.  Please remove it.

Reply:
removed opaeuio